src/components/icon-toolbar/icon-toolbar.test.js

```diff
@@ -1,6 +1,45 @@
 import React from 'react';
-import IconToolbar, { mapStateToProps, mapDispatchToProps } from './index';
+import IconToolbar, {
+  ThemeButton,
+  LabelButton,
+  mapStateToProps,
+  mapDispatchToProps
+} from './index';
 import { mockState, setup } from '../../utils/state.mock';
+import { getInitialState } from '../app/load-data';
+import formatData from '../../utils/format-data';
+import loremIpsum from '../../utils/data/lorem-ipsum.mock';
+
+describe('ThemeButton', () => {
+  it('toggles the theme on button click', () => {
+    let theme = 'dark';
+    const onToggle = t => {
+      theme = t;
+    };
+    const wrapper = setup.shallow(ThemeButton, { theme, onToggle });
+    wrapper.find('button').simulate('click');
+    expect(theme).toBe('light');
+  });
+});
+
+describe('LabelButton', () => {
```

Comment:
Do you think these components should be separated out of the icon-toolbar? Right now it's in the same file which means the tests are testing multiple components in one file rather than being focused

Reply:
I don't have a problem with multiple components in a single file. They're pretty simple, dumb components, it made sense to me to keep them together